### PR TITLE
make qa_crowbarsetup sourceable and drop duplicate code

### PR DESCRIPTION
**Pull request by bmwiedemann**

### CODE REVIEW

File: scripts/mkcloud [hunk 1]
```diff
@@ -72,6 +72,7 @@ nodenumbercomputedefault=2
 # expect to have this many physical machines attached via $mkch_physcloudif
 # these replace virtual machines, so we start less VMs
 : ${nodenumberphys:=0}
+. $qa_crowbarsetup
```
Comment (MaximilianMeister) [inline scripts/mkcloud]: i think this is a bit hidden. maybe we can add a source function which will have all sourcing code. as we are sourcing the libs in another place. i would put that together.

File: scripts/qa_crowbarsetup.sh [hunk 2]
```diff
@@ -70,6 +71,8 @@ rubyjsonparse()
         $1"
 }
 
+setcloudnetvars()
+{
 if [ -z "$cloud" ] ; then
```
Comment (MaximilianMeister) [inline scripts/qa_crowbarsetup.sh]: this needs some indentation

**Comment by MaximilianMeister** [comment]
:+1:

**Comment by jdsn** [comment]
:+1:

**Pull request merged**